exercises/word-search/canonical-data.json

```diff
@@ -1,13 +1,161 @@
 {
   "exercise": "word-search",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "comments": [
     "Grid rows and columns are 1-indexed.",
     "An expected value of -1 indicates that some sort of failure should occur."
   ],
   "cases": [
     {
-      "description": "Should locate words written left to right",
+      "description": "Should accept an initial game grid and a target search word",
+      "property": "search",
+      "grid": [
+        "jefblpepre"
+      ],
+      "wordsToSearchFor": [
+        "clojure"
+      ],
+      "expected": {
+        "clojure": -1
```

Comment:
why -1, as opposed to null? https://github.com/exercism/problem-specifications/blob/master/exercises/word-search/canonical-data.json#L672

Reply:
Oops!  That was a mistake.  Thanks for catching.